faux-mgs/src/main.rs

```diff
@@ -723,38 +724,74 @@ async fn run_command(
                 return Ok(Output::Json(serde_json::to_value(state).unwrap()));
```

Comment:
I don't know if @jclulow is consuming JSON for calling `state` yet, but it changes in this PR (even though this line didn't change). I'll let Josh say how painful that will be for him?

Reply:
@jclulow says we can do what we need to do :)